torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -166,6 +171,17 @@ struct TORCH_API KinetoThreadLocalState : public ProfilerThreadLocalState {
         cpu_trace->activities[idx].callStack = stacksToStr(kineto_events_[idx].stack());
 #endif
       }
+#ifdef USE_KINETO_UPDATED
+      if (kineto_events_[idx].hasTypes()) {
+        cpu_trace->activities[idx].addMetadata("Input type", dtypesToStr(kineto_events_[idx].dtypes());
```

Comment:
missing ')' here and below, we will remove USE_KINETO_UPDATED usage soon, so one more rebase will be needed